ansible/install_poetry.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+curl -sSL https://install.python-poetry.org | python3 -
+
+cd /opt/todoapp
+
+/home/ec2-user/.local/bin/poetry install
```

Comment:
It looks like you actually cover this anyway in your playbook later, so can we get rid of these steps?